bedrock/press/templates/press/press-inquiry.html

```diff
@@ -1,106 +1,88 @@
-            {% if form.user_email.errors %}
-              <p class="error-msg">{{_('Please enter a valid email address.')}}</p>
-            {% endif %}
-            {{ form.user_email }}
-          </div>
-
-          <div class="field required">
-            <label for="id_media_org">{{ _('Media Organization') }}</label>
-            {% if form.media_org.errors %}
-              <p class="error-msg">{{_('Please enter the name of your media organization.')}}</p>
-            {% endif %}
-            {{ form.media_org }}
-          </div>
-
-          <div class="field required">
-            <label for="id_inquiry">{{ _('Inquiry') }}</label>
-            {% if form.inquiry.errors %}
-              <p class="error-msg">{{_('Please enter the subject of your inquiry.')}}</p>
-            {% endif %}
-            {{ form.inquiry }}
-          </div>
-
-          <div class="field required">
-            <label for="id_deadline">{{ _('Deadline') }}</label>
-            {% if form.deadline.errors %}
-              <p class="error-msg">{{_('Please enter your deadline.')}}</p>
-            {% endif %}
-            {{ form.deadline }}
-            <div class="field-note">{{ _('2015-03-14 or March 14, 2015') }}</div>
-          </div>
+    <label for="id_name">{{ _('Name') }}</label>
+    {% if form.name.errors %}
+    <strong class="error-msg">{{_('Please enter your name.')}}</strong>
+    {% endif %}
+    {{ form.name }}
 
-          {{ form.office_fax }}
+    <label for="id_user_email">{{ _('Email Address') }}</label>
+    {% if form.user_email.errors %}
+    <strong class="error-msg">{{_('Please enter a valid email address.')}}</strong>
+    {% endif %}
+    {{ form.user_email }}
 
-        </fieldset>
+    <label for="id_media_org">{{ _('Media Organization') }}</label>
+    {% if form.media_org.errors %}
+    <strong class="error-msg">{{_('Please enter the name of your media organization.')}}</strong>
+    {% endif %}
+    {{ form.media_org }}
 
-        <div class="form-submit">
-          <input type="submit" class="button" id="form-submit" value="{{_('Submit')}}">
-        </div>
-      </form>
-    {% else %}
-      <h2>{{ _('Your request has been sent.') }}</h2>
-      <p>{{ _('Thank you for your inquiry.') }}</p>
+    <label for="id_inquiry">{{ _('Inquiry') }}</label>
+    {% if form.inquiry.errors %}
+    <strong class="error-msg">{{_('Please enter the subject of your inquiry.')}}</strong>
     {% endif %}
-    </div>
+    {{ form.inquiry }}
 
-    {% include 'press/includes/resources.html' %}
+    <label for="id_deadline">{{ _('Deadline') }}</label>
+    {% if form.deadline.errors %}
+    <strong class="error-msg">{{_('Please enter your deadline.')}}</strong>
+    {% endif %}
+    {{ form.deadline }}
+
+    {{ form.office_fax }}
+  </fieldset>
+  <button type="submit" id="form-submit" class="mzp-c-button mzp-t-primary">{{_('Submit')}}</button>
+</form>
+{% else %}
+<aside class="mzp-c-notification-bar mzp-t-success">
```

Comment:
Could the success page perhaps set a min-height, seeing as there's not much content?

![image](https://user-images.githubusercontent.com/400117/69133640-28019100-0aae-11ea-8e04-5633e8205a9c.png)